src/view/configuration/personalizationSection.jsx

```diff
@@ -90,6 +99,14 @@ const PersonalizationSection = ({ instanceFieldName }) => {
             copyToClipboard(prehidingSnippet);
           }}
         />
+        <FormikCheckbox
```

Comment:
@jonsnyder Do you think this deserves a Beta flag as well?

Reply:
👍 Yes